src/Tribe/Admin/Help_Page.php

```diff
@@ -0,0 +1,451 @@
+		 * @var object The plugin object, check `$this->get_plugins()` for more info
+		 */
+		return (object) apply_filters( 'tribe_help_api_data', $data, $plugin );
+	}
+
+	/**
+	 * Prints the Plugin box for the given plugin
+	 *
+	 * @param  string $plugin Plugin Name key
+	 * @return void
+	 */
+	public function print_plugin_box( $plugin ) {
+		$plugin = (object) $this->get_plugins( $plugin, false );
+		$api_data = $this->get_plugin_api_data( $plugin );
+		$addons = $this->get_addons( $plugin->name );
+		$plugins = get_plugins();
+
+		if ( $api_data ) {
+			if ( ! function_exists( 'install_plugin_install_status' ) ) {
+				include_once ABSPATH . '/wp-admin/includes/plugin-install.php';
+			}
+			$status = install_plugin_install_status( $api_data );
+			$plugin_active = is_plugin_active( $status['file'] );
+			$plugin_exists = isset( $plugins[ $status['file'] ] );
+
+			if ( 'install' !== $status['status'] && ! $plugin_active ){
+				$args = array(
+					'action' => 'activate',
+					'plugin' => $status['file'],
+					'plugin_status' => 'all',
+					'paged' => 1,
+					's' => '',
+				);
+				$activate_url = wp_nonce_url( add_query_arg( $args, 'plugins.php' ), 'activate-plugin_' . $status['file'] );
+				$link = '<a class="button" href="' . $activate_url . '" aria-label="' . esc_attr( sprintf( __( 'Activate %s', 'tribe-common' ), $plugin->name ) ) . '">' . esc_html__( 'Activate Plugin', 'tribe-common' ) . '</a>';
+			} elseif ( 'update_available' === $status['status'] ) {
+				$args = array(
+					'action' => 'upgrade-plugin',
+					'plugin' => $status['file'],
+				);
+				$update_url = wp_nonce_url( add_query_arg( $args, 'update.php' ), 'upgrade-plugin_' . $status['file'] );
+
+				$link = '<a class="button" href="' . $update_url . '">' . esc_html__( 'Upgrade Plugin', 'tribe-common' ) . '</a>';
+			} elseif ( $plugin_exists ) {
+				$link = '<a class="button disabled">' . esc_html__( 'You are up to date!', 'tribe-common' ) . '</a>';
+			}
+		}
+
+		if ( ! isset( $link ) ){
+			if ( $api_data ) {
+				$args = array(
+					'tab' => 'plugin-information',
+					'plugin' => $plugin->name,
+					'TB_iframe' => true,
+					'width' => 772,
+					'height' => 600,
+				);
+				$iframe_url = add_query_arg( $args, admin_url( '/plugin-install.php' ) );
+				$link = '<a class="button thickbox" href="' . $iframe_url . '" aria-label="' . esc_attr( sprintf( __( 'Install %s', 'tribe-common' ), $plugin->name ) ) . '">' . esc_html__( 'Install Plugin', 'tribe-common' ) . '</a>';
+			} else {
+				$link = null;
+			}
+		}
+		?>
+		<div class="tribe-help-plugin-info">
+			<h3><a href="<?php echo esc_url( $plugin->repo ); ?>"><?php echo esc_html( $plugin->title ); ?></a></h3>
+
+			<?php
+			if ( ! empty( $plugin->description ) && ! $plugin->is_active ) {
+				echo wpautop( $plugin->description );
+			}
+			?>
+
+			<?php if ( $api_data ) { ?>
+			<div>
+				<b><?php esc_html_e( 'Latest Version:', 'tribe-common' ); ?></b> <?php echo esc_html( $api_data->version ); ?>
```

Comment:
I feel like this `<div>`/`<b>`/`<br>` listing of meta information would be better presented using `<dl>`/`<dt>`/`<dd>` elements.